drivers/bluetooth/hci/h4.c

```diff
@@ -87,6 +89,13 @@ static inline void h4_get_type(void)
 		rx.remaining = sizeof(rx.acl);
 		rx.hdr_len = rx.remaining;
 		break;
+	case H4_ISO:
+		if (IS_ENABLED(CONFIG_BT_ISO)) {
+			rx.remaining = sizeof(rx.iso);
+			rx.hdr_len = rx.remaining;
+			break;
+		}
+	/* Fallthrough */
```

Comment:
Is this still alright? I remember seeing some changes to this annotation in the tree.

Reply:
I guess you are talking about the use of __fallthrough, I just converted these comments to use that instead.